airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -94,11 +99,12 @@ class JenkinsJobTriggerOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self,
-                 jenkins_connection_id,
-                 job_name,
-                 parameters="",
-                 sleep_time=10,
-                 max_try_before_job_appears=10,
+                 jenkins_connection_id: str,
+                 job_name: str,
+                 parameters: str = "",
```

Comment:
```suggestion
                 parameters: Optional[Union[str, Dict]] = None,
```

Reply:
Actually, when you look at the "build_job_url" method code and docstring, it should be Optional[Union[str, Dict, List]]

Reply:
See latest update.